src/org/zaproxy/zap/GuiBootstrap.java

```diff
@@ -345,48 +344,73 @@ private static void setDefaultViewLocale(Locale locale) {
      * {@link java.net.InetAddress InetAddress}) preventing some ZAP options from being correctly applied.
      */
     private void setupLookAndFeel() {
-        if (lookAndFeelSet) {
-            return;
-        }
-        lookAndFeelSet = true;
-
-        String lookAndFeelClassname = System.getProperty("swing.defaultlaf");
-        if (lookAndFeelClassname != null) {
-            try {
-                UIManager.setLookAndFeel(lookAndFeelClassname);
-                return;
-            } catch (final UnsupportedLookAndFeelException
-                     | ClassNotFoundException
-                     | ClassCastException
-                     | InstantiationException
-                     | IllegalAccessException e) {
-                logger.warn("Failed to set the specified look and feel: " + e.getMessage());
-            }
-        }
-
-        try {
-            // Set the systems Look and Feel
-            UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
-
-            if (Constant.isMacOsX()) {
-                OsXGui.setup();
-            } else {
-                // Set Nimbus LaF if available
-                for (final LookAndFeelInfo info : UIManager.getInstalledLookAndFeels()) {
-                    if ("Nimbus".equals(info.getName())) {
-                        UIManager.setLookAndFeel(info.getClassName());
-                        break;
-                    }
-                }
-            }
-        } catch (final UnsupportedLookAndFeelException
-                 | ClassNotFoundException
-                 | InstantiationException
-                 | IllegalAccessException e) {
-            logger.warn("Failed to set the \"default\" look and feel: " + e.getMessage());
-        }
+    	if(lookAndFeelSet) {
+    		
+    		return;
+    	}
+    	
+    	String lookAndFeelClassnameFromCommandline = System.getProperty("swing.defaultlaf");
+    	    	
+    	if( lookAndFeelClassnameFromCommandline!= null) {
+        	try{
+        			UIManager.setLookAndFeel(lookAndFeelClassnameFromCommandline);
+        			lookAndFeelSet = true;
+        			return;
+        	} catch (final UnsupportedLookAndFeelException
+        			| ClassNotFoundException
+        			| ClassCastException
+        			| InstantiationException
+        			| IllegalAccessException e) {
+        		logger.warn("Failed to set the specified look and feel:" + e.getMessage());
+        	}
+        }else { 
```

Comment:
It should not have an `else` statement, we want the following to be picked if the above fails (to keep the same behaviour).